### DISCUSSION TITLE
automatic translation in google Chrome from English, although the Polish language is set in the application

### DISCUSSION THREAD

**Discussion by dawnog** [globaleaks/globaleaks-whistleblowing-software#4183] [Q&A]
The Google Chrome app offers an English translation even though Polish is set in the app.

In addition, it does this incorrectly and is misleading.

I found something like this written in the application code and I cannot change it
html lang=‘en’.
https://github.com/globaleaks/GlobaLeaks/blob/7d9d6ace31e222f4196311b395f1093695553de0/client/app/index.html#L2C1-L2C17
And because of this Chrome treats it as if it were an English page. In the documentation and settings of GlobaLeaks I unfortunately did not find the option to add custom html tags.

Please help
Dawid

**Comment by evilaliv3** [comment]
Thank you.

Unfortunately for accessibility purposes a language should be expressed in the html page. 
The language is then updated if you configure the platform to be in polish.

If you would like to pass me the link of your page i may try to understand the problem.
